### ISSUE TITLE
`set`, `string` and `source_group` not highlighted correctly

### ISSUE THREAD

**Issue by adtennant** [lucas-clemente/language-cmake#3]
Calls to `set(...)`, `string(...)` and `source_group(...)` only have the `s` at the start highlighted correctly, the other characters (`et`, `tring`, `ource_group`) are not highlighted.

This may affect other CMake functions (likely all beginning with s), but I've not had a chance to test them.

**Comment by sryze** [comment]
:+1: I noticed this too. The keyword regex in [`cnake listfile.cson`](https://github.com/lucas-clemente/language-cmake/blob/master/grammars/cmake%20listfile.cson) is not exactly human editable (seems to be generated by a tool), otherwise this would be trivial to fix...

**Comment by lucas-clemente** [comment]
The grammar file is the result of converting the [cmake textmate package](https://github.com/textmate/cmake.tmbundle) using atoms [built-in converter](https://atom.io/docs/v0.192.0/hacking-atom-converting-from-textmate). Unfortunately I don't either understand how atom's grammar works.

I've tried regenerating it from the textmate bundle, and the result looks quite different from the current master (but doesn't fix this issue). I've pushed it as the `new-conversion` branch.
